tests/requirements.txt

```diff
@@ -12,7 +12,7 @@ menuinst >=2
 packaging
 pkginfo
 psutil
-py-lief
+py-lief <0.15
```

Comment:
#5565 should make this unnecessary

Reply:
Depending on when PRs get reviewed happy to remove this change or switch up the merge order as is convenient.

Reply:
Likewise, I could merge or rebase on main and undo this inside #5565 if this gets merged first. Defer to maintainers on what order they'd rather merge in.

Reply:
I removed this pinning now that your PR is in. Thanks!